src/test/infra/lttng/test_latency_definitions.py

```diff
@@ -1495,6 +1621,54 @@ def _create_timer_struct(
 
         return _create_timer_struct
 
+    def test_var_pass_empty_data(
+        self,
+        mocker,
+        create_lttng,
+        create_callback_lttng,
+        create_callback_struct,
+        bridge_mock,
+        bridge_setup_get_callback,
+    ):
+        callback_obj = 5
+        callback_obj_ = 7
+        # pid, tid = 15, 16
```

Comment:
This seems like an unnecessary description.

Reply:
addressed in :[c670001](https://github.com/tier4/CARET_analyze/pull/210/commits/c6700010b36b4d8e0ec50486f0f6b972bd11a59a)